packages/map/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "learn-starter",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build && next export -o map",
+    "start": "next start",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|vue)\"",
+    "convert": "node scripts/pointToPolygon.js"
+  },
+  "dependencies": {
+    "@turf/circle": "^6.5.0",
+    "@turf/transform-rotate": "^6.5.0",
+    "@zeit/next-css": "^1.0.1",
+    "csvtojson": "^2.0.10",
+    "intersection-observer": "^0.12.0",
+    "mapbox-gl": "^1.13.1",
+    "next": "9.5.4",
+    "next-images": "^1.7.0",
+    "react": "^16.14.0",
```

Comment:
`react` and `react-dom` are already included in the parent `package.json` do they should not be duplicated here